snap/snapcraft.yaml

```diff
@@ -1,5 +1,6 @@
 name:               krop
-version:            0.4.13.1
+version:            0.5.1
```

Comment:
Why the version is jumping to `5.1` ? 
See best practices on <https://semver.org/>

Reply:
I tried using 0.4.13.1, but it wouldn't work, so instead, I tried using the one from upstream

Reply:
I remember replying here few months back, but I thought I did.  Sorry for ghosting you.

Reply:
Let me know if you managed to get 0.14.13.1 to work.

Reply:
You might need to use `"0.14.13.1"` instead of `0.14.13.1`

Not sure what the version of Krop is but in general, the snap `version` should follow the version of the software itself. The snap `revision` is automatically set and tracks the "version of the snap".